src/vellum/workflows/nodes/displayable/bases/utils.py

```diff
@@ -103,6 +103,12 @@ def primitive_to_vellum_value(value: Any) -> VellumValue:
 
 
 def primitive_to_vellum_value_request(value: Any) -> VellumValueRequest:
+    if hasattr(value, "__class__") and value.__class__.__name__ == "WorkflowInputReference":
+        from vellum.workflows.references import WorkflowInputReference
+
+        if isinstance(value, WorkflowInputReference):
+            return StringVellumValueRequest(value=value.name)
```

Comment:
We shouldn't need to make a special exception here for `WorkflowInputReference`. I think we'll want to solve this at the node's serialization logic